src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+
+        Square square1 = new Square(5.0, Color.RED);
+        square1.draw();
+        IsoscelesTrapezoid isoscelesTrapezoid1 = new IsoscelesTrapezoid(10, 10, 20, Color.GREEN);
+        isoscelesTrapezoid1.draw();
+
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        Figure[] figures = new Figure[3];
+        figures[0] = figureSupplier.getRandomRectangle();
+        figures[1] = figureSupplier.getRandomCircle();
+        figures[2] = figureSupplier.getRandomRightTriangle();
```

Comment:
Don't use method names like `getRandomRectangle`, `getRandomCircle`, `getRandomRightTriangle` as they suggest specific figure types which contradicts with the idea of having a single `getRandomFigure` method that returns a random figure. Consider refactoring these methods into a single `getRandomFigure` method that utilizes a switch or if-else construct to return different figures.